app/(main)/feed.js

```diff
@@ -160,7 +160,6 @@ export default function FeedPage() {
 						/>
 					)}
 					keyExtractor={(item) => item[0]}
-					extraData={refreshing}
```

Comment:
I thought this was necessary? for reloading

Reply:
it is, but it doesnt work right now so i removed it incase it causes any unexpected behavior